RELEASE/scripts/autoscend/quests/level_any.ash

```diff
@@ -697,3 +697,141 @@ boolean LX_meatMaid()
 
 	return false;
 }
+
+item LX_getDesiredWorkshed(){
+	string currentWorkshed = get_property("auto_workshed").to_lower_case();
+	//return the actual item name in case a shorthand is used
+	switch(currentWorkshed)
+	{
+		case "model train set":
+		case "train":
+			return $item[model train set];
+		case "cold medicine cabinet":
+		case "cmc":
+			return $item[cold medicine cabinet];
+		case "asdon martin keyfob":
+		case "asdon":
+			return $item[Asdon Martin keyfob];
+		case "diabolic pizza cube":
+		case "pizza":
+			return $item[diabolic pizza cube]; //unless support is added, don't want to use this
+		case "portable mayo clinic":
+		case "mayo":
+			return $item[portable mayo clinic];
+		case "little geneticist dna-splicing lab":
+		case "dnalab":
+			return $item[little geneticist dna-splicing lab];
+		//passive worksheds
+		case "snow machine":
+			return $item[snow machine]; //but you need a garden
+		case "warbear auto-anvil":
+			return $item[warbear auto-anvil];
+		case "warbear chemistry lab":
+			return $item[warbear chemistry lab];
+		case "warbear high-efficiency still":
+			return $item[warbear high-efficiency still];
+		case "warbear induction oven":
+			return $item[warbear induction oven];
+		case "warbear jackhammer drill press":
+			return $item[warbear jackhammer drill press]; //We very rarely pulverize things but if someone really wants to use it, sure they can select it
+		case "warbear lp-rom burner":
+			return $item[warbear lp-rom burner]; //If someone really wants to record some AT buffs on their own, allow them to select it
+		case "spinning wheel":
+			return $item[spinning wheel]; //If someone really wants additional meat. They will need to use it on their own
+		case "auto":
+		default:
+			// auto_workshed is invalid or none/false/whatever to say don't do this
+			return $item[none];
+	}
+}
+
+boolean LX_setWorkshed(){
+	item desiredShed = LX_getDesiredWorkshed();
+	item existingShed = get_workshed();
+	boolean workshedChanged = get_property("_workshedItemUsed");

```

Comment:
Have you tested `workshedChanged` is being set properly? Typically we add a to_boolean in there to explicitly state what type it is. Like:
`boolean workshedChanged = get_property("_workshedItemUsed").to_boolean();`

Reply:
ah looks like the build is failing for this, the to_boolean is needed.

A trick to test this locally is to enter `validate autoscend` in the cli. It will tell ya about any syntax and general compile issues